src/mbgl/mtl/drawable.cpp

```diff
@@ -387,7 +387,7 @@ void Drawable::bindUniformBuffers(RenderPass& renderPass) const noexcept {
             const auto& block = uniformBlocks.get(id);
             if (!block) continue;
             const auto& uniformBuffer = getUniformBuffers().get(id);
-            assert(uniformBuffer && "UBO missing, drawable skipped");
+            // assert(uniformBuffer && "UBO missing, drawable skipped");
```

Comment:
can this be reenabled?

Reply:
No, because now some UBO can miss at the drawable level, being set a layer group level. So I will remove this assert.